hermes-mock/src/main/java/pl/allegro/tech/hermes/mock/HermesMockHelper.java

```diff
@@ -80,6 +80,16 @@ public void addStub(String topicName, Response response, String contentType) {
                         .withFixedDelay(toIntMilliseconds(response.getFixedDelay())))
         );
     }
+    public void addStub(String topicName, Response response, String contentType, AvroMatchesPattern pattern) {
```

Comment:
If you would apply changes mentioned earlier, this method will accept `ValueMatcher<com.github.tomakehurst.wiremock.http.Request> valueMatcher` instead of `AvroMatchesPattern pattern`

Reply:
If you will have import conflicts with `Request` and `Response` classes from `pl.allegro.tech.hermes.mock` package, you can move two of them to `pl.allegro.tech.hermes.mock.exchange` package :)